src/components/Jobs/components/Job/Job.js

```diff
@@ -38,6 +39,7 @@ import { jobExecutionPropTypes } from './jobExecutionPropTypes';
 
 import * as API from '../../../../utils/upload';
 import * as CompositeJobFields from '../../../../utils/compositeJobStatus';
+import { cancelMultipartJob } from '../../../../utils';
```

Comment:
Consolidate/fix these imports, there's multiple impotrs from `utils` and it seems like some are weirdly indented?